tests/ruby/RubySecurityBearTest.py

```diff
@@ -0,0 +1,45 @@
+import os.path
+import unittest
+from queue import Queue
```

Comment:
make it in alphabeticcal order.

Reply:
I took ref from https://github.com/coala/coala-bears/blob/master/tests/c_languages/CPPCheckBearTest.py#L3.

Reply:
I took ref from https://github.com/coala/coala-bears/blob/master/tests/c_languages/CPPCheckBearTest.py#L3.

Reply:
ohh :P cc @Makman2